src/keypath.js

```diff
@@ -70,14 +70,15 @@ function pathToArray(path) {
  *
  * @param {Object} obj The object scope
  * @param {String|Array} path The path of the property to retrieve
+ * @param {mixed} defaultValue The default value returned if path was not found. Default is undefined.
```

Comment:
🤔 mixed really looks like a PHP thing 😅 I would prefer * or any. BTW thanks @bato for the PR!!

Reply:
Ehm... you are right :smile_cat: